itez/templates/includes/profile-photo-update.html

```diff
@@ -0,0 +1,86 @@
+
```

Comment:
this file is also called profile update why do we have 2 duplicate files to do the  same thing?